lib/gateways/logging_api.rb

```diff
@@ -0,0 +1,14 @@
+module Gateways
+  class LoggingApi
+    LOGGING_API_SEARCH_ENDPOINT = SITE_CONFIG['logging_api_search_endpoint'].freeze
+
+    def search(username:)
+      return [] if username.nil? || username.empty?
```

Comment:
Probably also validate beyond nil, and ensure a username is 6 characters long before we send this request over the wire.